models/api.models.js

```diff
@@ -86,3 +86,12 @@ return rows[0]
 })
 }
 
+exports.deleteDbApiComment=(comment_id)=>{
+    return db.query(`DELETE from comments WHERE comment_id=$1 RETURNING *`,[comment_id])
+    .then(({rows})=>{
+        if(rows.length ===0){
+            return Promise.reject({status:404,msg:"comment_id not found"})
+        }
```

Comment:
Nice specific message on the error handling - good to be specific when we can be, good to be generic when we want re-usability 🥇